tox.ini

```diff
@@ -37,6 +39,11 @@ install_command= python -m pip install --no-cache-dir {opts} {packages}
 basepython= python3.8
 pip_pre= true
 
+[testenv:warnings]
+basepython= python3.8
+commands=
+    pytest --remote-data -W error -W ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.asdftypes
```

Comment:
Can the `-W ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.asdftypes` be removed here as we capture and ignore it elsewhere?

Reply:
Or does this override the `filterwarnings` directive in `[tool:pytest]`?

Reply:
Right, that's the issue -- pytest uses the last matched rule to determine if the warning should be ignored, and `filterwarnings` seems to get loaded first.